Chaos/ChampionDrakathMerge.cs

```diff
@@ -0,0 +1,161 @@
+        Core.SetOptions();
+
+        BuyAllMerge();
+        Core.SetOptions(false);
+    }
+
+    public void BuyAllMerge(string? buyOnlyThis = null, mergeOptionsEnum? buyMode = null)
+    {
+        LOC.Hero();
+        //Only edit the map and shopID here
+        Adv.StartBuyAllMerge("championdrakath", 2055, findIngredients, buyOnlyThis, buyMode: buyMode);
+
+        #region Dont edit this part
+        void findIngredients()
+        {
+            ItemBase req = Adv.externalItem;
+            int quant = Adv.externalQuant;
+            int currentQuant = req.Temp ? Bot.TempInv.GetQuantity(req.Name) : Bot.Inventory.GetQuantity(req.Name);
+            if (req == null)
+            {
+                Core.Logger("req is NULL");
+                return;
+            }
+
+            switch (req.Name)
+            {
+                default:
+                    bool shouldStop = Adv.matsOnly ? !dontStopMissingIng : true;
+                    Core.Logger($"The bot hasn't been taught how to get {req.Name}." + (shouldStop ? " Please report the issue." : " Skipping"), messageBox: shouldStop, stopBot: shouldStop);
+                    break;
+                #endregion
+
+                case "Drakath Armor":
+                        DAB.DrakathArmorQuest();
+                    break;
+
+                case "Champion Drakath Insignia":
+                    Core.FarmingLogger(req.Name, quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.Logger("You have to kill Champion Drakath weekly to get his insignias, use your army to kill it easily");
+                        return;
+                    }
+                    break;
+
+                case "Original Drakath Armor":
+                        DAB.DrakathOriginalArmor();
+                    break;
+
+                case "Blade of Chaos":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.EquipClass(ClassType.Solo);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("ultradrakath", "Champion of Chaos", req.Name, quant, false, false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    break;
+
+                case "Chaos Avenger's Greatsword":
+                case "Chaos Avenger Armor":
+                    Core.FarmingLogger(req.Name, quant);
+                    
+                    CAV.GetClass();
+                    Core.EquipClass(ClassType.Solo);
+                    
+                    if (Core.isCompletedBefore(8301))
```

Comment:
the if statement here could be put before line 51/52 with a return after:

`
if (!Core.isCompletedBefore(8301))
return;
`

you wont need this here then and can just do the buy line 👍